psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -1347,15 +1347,15 @@ void OEProp::compute_quadrupole(bool transition)
 
     // Print multipole components
     double dfac = pc_dipmom_au2debye * pc_bohr2angstroms;
-    outfile->Printf( "  %sQuadrupole Moment: (Debye Ang)\n", (transition ? "Transition " : ""));
+    outfile->Printf( "  %sQuadrupole Moment: [D Å]\n", (transition ? "Transition " : ""));
```

Comment:
Doesn't the funny A cause weird output on some terminals?

Reply:
Drat, I thought I had run my non-ascii checker over the codebase again. I don't know about users, but sphinx and conda really don't like non-ascii.

Thanks, I'll fix it. Perils of reordering and splitting all your commits.